src/optionals/object/object.c

```diff
@@ -48,6 +48,17 @@ static Value objectGetClassRefInternal(DictuVM *vm, int argCount, Value *args) {
     return objectGetClassRefImpl(vm, argCount, args, true);
 }
 
+static Value objectHash(DictuVM *vm, int argCount, Value *args) {
+    if (argCount != 1) {
+        runtimeError(vm, "hash() takes 1 argument (%d given)", argCount);
+        return EMPTY_VAL;
+    }
+
+    char str[64];
+    sprintf(str, "%llu", (unsigned long long)args[0]);
+    return OBJ_VAL(copyString(vm, (char *)str, 64));
```

Comment:
```suggestion
    char str[21];
    snprintf(str, 21, "%llu", (unsigned long long)args[0]);
    return OBJ_VAL(copyString(vm, (char *)str, 20));
```

Although we're dealing with 64 bits the max size for a uint64_t is 18,446,744,073,709,551,615, so will be 20 chars